module/VuFind/src/VuFind/Marc/Serialization/Iso2709.php

```diff
@@ -0,0 +1,185 @@
+<?php
+/**
+ * ISO2709 MARC exchange format support class.
+ *
+ * PHP version 7
+ *
+ * Copyright (C) The National Library of Finland 2020.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  MARC
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:record_drivers Wiki
+ */
+namespace VuFind\Marc\Serialization;
```

Comment:
Might be worth defining a SerializationInterface both as a clue to help with future serialization implementations and to allow for stronger typing if we pass serializations around in the future (for now, I'm fine with keeping these static, but if we want this to be more extensible, we might rethink that so we can inject the serializations into the reader).

Reply:
Interface added. I thought about the capability to inject serializations, but since we always need ISO2709 and MARCXML (at least the export support even if the internal format was something else), I deemed it unnecessarily complex or full of magic for now. I also wanted to make sure we don't need to instantiate too many classes just to check which one we need.

Reply:
Makes sense!